velox/connectors/hive/storage_adapters/abfs/AbfsConfig.h

```diff
@@ -28,17 +29,33 @@ namespace facebook::velox::filesystems {
 // This is used to specify the Azurite endpoint in testing.
 static std::string kAzureBlobEndpoint{"fs.azure.blob-endpoint"};
 
+// The authentication mechanism is set in `fs.azure.account.auth.type` (or the
+// account specific variant). The supported values are SharedKey, OAuth and SAS.
+static std::string kAzureAccountAuthType{"fs.azure.account.auth.type"};
+
+static std::string kAzureAccountKey{"fs.azure.account.key"};
```

Comment:
We also need to fix the config names to be consistent with other Velox filesystems.
can we use `hive.abfs` prefix instead of `fs.azure`?

Reply:
These are user-facing configurations that have consistently been referred to by these names. Using a different prefix might cause confusion for users.